### PR TITLE
Tweak: Add `em` unit to Word Spacing control in Text Path widget

### PR DESCRIPTION
**Pull request by rodolphebertozzo**

### CODE REVIEW

**Comment by rodolphebertozzo** [comment]
@StyleShit

**Comment by rodolphebertozzo** [comment]
@rami-elementor

File: modules/shapes/widgets/text-path.php [hunk 1]
```diff
@@ -344,13 +344,16 @@ protected function register_style_tab() {
...
-				'size_units' => [ 'px' ],
+				'size_units' => [ 'px', 'em' ],
 				'range' => [
 					'px' => [
 						'min' => -20,
 						'max' => 20,
 						'step' => 1,
 					],
+					'em' => [
+						'step' => 0.1,
```
Comment (rami-elementor) [inline modules/shapes/widgets/text-path.php:355]:
```suggestion
						'min' => -1,
						'max' => 1,
						'step' => 0.1,
```

I've tested the PR. Looks good!

But in this scope, without the `min` / `max` it's not usable.

Please add the min/max values. I've tested with other values, but between `-1` to `1` feels like they are the best.

Comment (rodolphebertozzo) [reply modules/shapes/widgets/text-path.php:357]: Done.

**Review by rami-elementor — APPROVED** [review]

**Review by arielk — APPROVED** [review]

**Pull request merged**